### PR TITLE
Prevent empty pickers from throwing

### PR DESCRIPTION
**Pull request by karlhungus**
Dependant on: https://github.com/marvinlabs/android-floatinglabel-widgets/pull/25

Addresses: https://github.com/marvinlabs/android-floatinglabel-widgets/issues/21

Not sure if this is a reasonable solution to @dsdebastiani issue -- also not sure if doing nothing is reasonable.

### CODE REVIEW

**Comment by karlhungus** [comment]
FYI @sebmartin @pseudomuto

**Comment by karlhungus** [comment]
Oops forgot to ping you @vpratfr

File: demo/build.gradle [hunk 1]
```diff
@@ -27,7 +27,7 @@ android {
 
     buildTypes {
         release {
-            runProguard true
```
Comment (pseudomuto) [inline demo/build.gradle]: was this removed intentionally?

File: demo/src/main/res/layout/fragment_item_widgets.xml [hunk 2]
```diff
@@ -30,5 +30,11 @@
...
             app:flw_labelText="Single item picker" />
 
+        <com.marvinlabs.widget.floatinglabel.itempicker.FloatingLabelItemPicker
+            android:id="@+id/picker3"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="15dp"
+            app:flw_labelText="Empty item picker" />
     </LinearLayout>
 </ScrollView>
```
Comment (pseudomuto) [inline demo/src/main/res/layout/fragment_item_widgets.xml]: missing new line here

**Comment by pseudomuto** [comment]
Seems pretty straightforward. LGTM :+1:

File: demo/build.gradle (hunk 1 above)
Comment (karlhungus) [inline demo/build.gradle]: yeah, previous pr -- http://tools.android.com/tech-docs/new-build-system/migrating-to-1-0-0

**Comment by karlhungus** [comment]
@vpratfr This one worth merging (if so i'll rebase on latest master)?

**Comment by vpratfr** [comment]
Not sure i would fix it like that. Is this one urgent?

**Comment by karlhungus** [comment]
Not urgent, I've gotten around it by hiding the empty widget, I think
throwing an exception for this case is surprising  though.

**Comment by karlhungus** [comment]
Going to close this as seems like it's not the preferred solution and not necessary for my needs. If you want it back let me know i'll bring it up to date.

**Pull request closed without merging**